### ISSUE TITLE
U2F keys not adding

### ISSUE THREAD

**Issue by rroseselavy42** [Ylianst/MeshCentral#197]
Hey There, 
I didn't see another issue on this.
When trying to add a hardware key to an account I'm getting this series of errors. 
```
-------- 4/22/2019, 12:34:55 PM ---- 0.3.2-r --------

(node:12644) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at /opt/meshcentral/node_modules/meshcentral/meshuser.js:2216:59
    at processTicksAndRejections (internal/process/task_queues.js:86:5)


-------- 4/22/2019, 12:34:55 PM ---- 0.3.2-r --------

(node:12644) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:12644) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

```

**Comment by Ylianst** [comment]
Interesting, what version of Node?

**Comment by Ylianst** [comment]
Published MeshCentral v0.3.2-w with fixes for this. Let me know if it works.

**Comment by rroseselavy42** [comment]
all working! Thanks so much!
